feed/feed.go

```diff
@@ -94,7 +95,17 @@ func PostsFromStorePosts(storePosts []gen.CandidatePost) []Post {
 	return posts
 }
 
-func chronologicalGenerator(requireTags []string, excludeTags []string) GenerateFunc {
+type chronologicalGeneratorOpts struct {
```

Comment:
Thanks for breaking this out into a struct, it was getting pretty hard to read in its current form anyway.